tests/clock/ClockUtils.py

```diff
@@ -0,0 +1,384 @@
+                logging.info(f'Command Error!\nError message: "{cmd_output}"')
+            logging.info(f'Output type: {type(cmd_output)}')
+            logging.info(f'Output: {cmd_output}')
+
+        with allure.step('Convert output to string'):
+            logging.info('Convert output to string')
+            cmd_output = str(cmd_output)
+            logging.info(f'Output type: {type(cmd_output)}')
+            logging.info(f'Output: {cmd_output}')
+        return cmd_output
+
+    @staticmethod
+    def parse_show_clock_output(show_clock_output):
+        """
+        @summary:
+            Split output of show clock into date, time and timezone strings
+
+            Exapmple:
+            "Mon 03 Apr 2023 11:29:46 AM UTC" -> {"date": "Mon 03 Apr 2023", "time": "11:29:46 AM", "timezone": "UTC"}
+        @param show_clock_output: the given show clock output
+        @return: The splited output as a dict
+        """
+        with allure.step('Split output of show clock'):
+            logging.info('Split output of show clock')
+            output_list = show_clock_output.split(' ')
+            date = ' '.join(output_list[0:4])
+            time = ' '.join(output_list[4:6])
+            timezone = ' '.join(output_list[6:])
+            logging.info(f'Splited output:\ndate: "{date}"\ntime: "{time}"\ntimezone: "{timezone}"')
+
+            res = {
+                ClockConsts.DATE: date,
+                ClockConsts.TIME: time,
+                ClockConsts.TIMEZONE: timezone
+            }
+            logging.info(f'res dict: {res}')
+
+            return res
+
+    @staticmethod
+    def parse_linux_cmd_output(linux_cmd_output):
+        """
+        @summary:
+            Parse output of a linux command.
+
+            Example:
+                timedatectl's output:
+                            "Local time: Tue 2023-04-04 08:22:01 MDT
+                           Universal time: Tue 2023-04-04 14:22:01 UTC
+                                 RTC time: Tue 2023-04-04 14:22:01
+                                Time zone: America/Inuvik (MDT, -0600)
+                System clock synchronized: no
+                              NTP service: n/a
+                          RTC in local TZ: no"
+
+                will become:
+                {
+                    "Local time": "Tue 2023-04-04 08:22:01 MDT",
+                    "Universal time": "Tue 2023-04-04 14:22:01 UTC",
+                    ...
+                }
+        @param linux_cmd_output: given output of a linux command (str)
+        @return: dictionary as mentioned in the example
+        """
+        with allure.step('Parse linux command output into dictionary'):
+            logging.info('Parse linux command output into dictionary')
+            rows = [row.strip() for row in linux_cmd_output.split('\n')]  # split by rows
+            logging.info(f'rows: {rows}')
+            res_dict = {}
+            for row in rows:
+                logging.info(f'row: "{row}"')
```

Comment:
Can be changed to debug.

Reply:
you mean to change to 'logging.debug(...)' ?